docs/usage/config-overview.md

```diff
@@ -155,6 +155,12 @@ Inherited config may use all Repository config settings, and any Global config o
 
 For information on how the Mend Renovate App supports Inherited config, see the dedicated "Mend Renovate App Config" section toward the end of this page.
 
+#### Presets handling
+
+If presets are present in the inherited config, they will be resolved and added to the inherited config prior to being merged on top of global config.
+
+Using `ignorePresets` in your repository config to ignore certain presets will not affect presets present in the inherited config, as inherited config is resolved before repository config.
```

Comment:
```suggestion
Using `ignorePresets` in your repository config to ignore certain presets will _not_ affect presets present in the inherited config.
This is because inherited config is resolved _before_ the repository config.
```

This is a difficult sentence. Let's start by splitting it into two smaller parts, and emphasizing the key points.

What are you trying to explain here? I think it is something like this?

> You can _not_ ignore inherited presets with `ignorePresets`, because inherited presets are resolved _before_ the repository config.